### PR TITLE
ADRs, Landing Page, UI Mockups & More

### PR DESCRIPTION
**Pull request by jeafreezy**
This PR introduces the approved ADR for the styling ramework.

cc: @kshitijrajsharma

### CODE REVIEW

File: frontend/pnpm-lock.yaml
Comment (kshitijrajsharma) [inline frontend/pnpm-lock.yaml:1]: Do we need lock file too ? @jeafreezy

File: frontend/docs/architecture/adr-choose-styling-library/adr2.md
Comment (kshitijrajsharma) [inline frontend/docs/architecture/adr-choose-styling-library/adr2.md:1]: @omranlm  This file is for you to review

**Review by kshitijrajsharma — APPROVED** [review]

File: frontend/pnpm-lock.yaml
Comment (jeafreezy) [reply frontend/pnpm-lock.yaml:1]: Yes, @kshitijrajsharma . It's typical in FE projects to have the lock files on GitHub to pin the dependencies.

**Comment by jeafreezy** [comment]
@omranlm you can grab this branch.

**Comment by jeafreezy** [comment]
UI mockups are here on [Figma](https://www.figma.com/design/1WXme5jfWV4tCaY9Rcaekk/fAIr-Project?node-id=0-1&node-type=canvas&t=1pxbORWzTEX11f35-0).

The UI design requires access, please reach out to @omranlm or @kshitijrajsharma for access.

**Comment by kshitijrajsharma** [comment]
Request to rebase and update the current master on the working branch so that it would be easy to review! @jeafreezy

**Comment by jeafreezy** [comment]
Please ignore the base switch. I was trying to observe the changes in the commit timeline.

**Review by kshitijrajsharma — APPROVED** [review]

**Pull request merged**